From 7137ea7c201c1f3d7f7c2feb749f5c311ee6ea6c Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Fri, 21 Apr 2017 13:31:57 +0200
Subject: [PATCH 3/5] udev: Add support for I2C Elan touchpads (without dmi)

This adds support for I2C Elan touchpads, such as the ones found in
various ARM CrOS devices. These devices do not use dmi.

The pressure range is copied as-is from the current Elantech PS/2
touchpads entry. It is not adapted to every touchpad configuration.

Signed-off-by: Paul Kocialkowski <contact at paulk.fr>
---
 udev/90-libinput-model-quirks.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/udev/90-libinput-model-quirks.hwdb b/udev/90-libinput-model-quirks.hwdb
index 4b9b7f2..77d0a67 100644
--- a/udev/90-libinput-model-quirks.hwdb
+++ b/udev/90-libinput-model-quirks.hwdb
@@ -88,6 +88,10 @@ libinput:name:*ETPS/2 Elantech Touchpad*:dmi:*
  LIBINPUT_ATTR_PRESSURE_RANGE=24:8
  LIBINPUT_MODEL_ELANTECH_TOUCHPAD=1
 
+libinput:name:*Elan Touchpad*
+ LIBINPUT_MODEL_ELANTECH_TOUCHPAD=1
+ LIBINPUT_ATTR_PRESSURE_RANGE=24:8
+
 ##########################################
 # Google
 ##########################################
-- 
2.12.2

